inst/stan/functions/spatial_rt_process_rng.stan

```diff
@@ -0,0 +1,39 @@
+/**
+  * Generates the subpopulation Rt trajectories for a spatially correlated model
+  * @param log_state_rt "State" level unadjusted Rt, on log scale.
+  * @param dist_matrix Distance matrix for use in correaltion function.
+  * @param sigma_eps Parameter for construction of covariance matrix
+  * of spatial epsilon.
+  * @param phi Parameter for construction of correaltion matrix.
+  * @param l Parameter for construction of correlation matrix.
+  * @param spatial_deviation_ar_coeff Coefficient for AR(1) temporal correlation on
+  * subpopulation deviations.
+  * @param spatial_deviation_init Initial vector of spatial deviation.
+  * @return A matrix for unadjusted Rt where rows are subpopulations
+  * and columns are time points.
+  */
+matrix spatial_rt_process_rng(vector log_state_rt,
```

Comment:
As above, should indicate in the function name that this is for an exponential decay correlation or rewrite to be more generic and accept `sigma_matrix` as a pre-computed argument.